tooling/autodispose-rx-error-prone-checker/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+  id "net.ltgt.errorprone" version "0.0.13"
+  id "com.github.johnrengelman.shadow" version "2.0.1"
+  id "com.github.johnrengelman.plugin-shadow" version "2.0.0"
+  id "java"
+}
+
+// we use this config to get the path of the JDK 9 javac jar, to
+// stick it in the bootclasspath when running tests
+configurations.maybeCreate("epJavac")
+
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+
+dependencies {
+  compileOnly deps.apt.autoService
+  testCompile deps.rx.java
+
+  compileOnly deps.build.errorProneCheckApi
+
+  errorprone deps.build.errorProne
+
+  testCompile deps.test.junit
+  testCompile(deps.build.errorProneTestHelpers) {
+    exclude group: "junit", module: "junit"
+  }
+  testCompile project(':autodispose')
+
+  epJavac deps.build.errorProneCheckApi
```

Comment:
Remove this line

Reply:
https://github.com/uber/AutoDispose/pull/156/commits/05a3a34b3778da0031e5dfa0161256f224ed6cdf